showcase/src/assets/examples/progress-indicator/progress-indicator.html

```diff
@@ -0,0 +1,6 @@
+<button class="progress-indicator-button" color="primary" mat-raised-button (click)="showProgressIndicator()">
+	<span translate>{{ show ? "SHOWCASE.DEMO.PROGRESS_INDICATOR.HIDE" : "SHOWCASE.DEMO.PROGRESS_INDICATOR.SHOW" }}</span>
```

Comment:
In the example files we don't put the translation keys but the english translation ;)